genericjpa/README.md

```diff
@@ -0,0 +1,84 @@
+Hibernate-Search-JPA
+====================
```

Comment:
Maybe the first few pages of the chapters 5,6 and 7 from here can help to understand why I did things like this and where this code comes from:

https://github.com/s4ke/Bachelor-Thesis/raw/Update-System-rework/tex/src/bachelor_thesis.pdf